### PR TITLE
bio-tbtools: add pkg

### PR DESCRIPTION
**Pull request by Kiri2002**
## Involved packages

 - bio-tbtools

## Involved issue

Close #

## Details

- [ ] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [x] Provide New Package
- [ ] Fix the Packages
  - [ ] PKGBUILD
  - [ ] lilac.yaml
  - [ ] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

**Comment by Kiri2002** [comment]
@starsareintherose after this merged, the old pkg tbtools should be removed by admin

File: BioArchLinux/bio-tbtools/LICENSE
Comment (starsareintherose) [inline BioArchLinux/bio-tbtools/LICENSE:1]: Should be deleted

**Review by starsareintherose — CHANGES_REQUESTED** [review]

**Comment by starsareintherose** [comment]
You are now member, should have ability to rm some folder

**Comment by starsareintherose** [comment]
You are not allowed to merge

**Comment by Kiri2002** [comment]
> You are not allowed to merge

sorry, i mis-understant

**Comment by starsareintherose** [comment]
Please re-commit pkg

**Comment by starsareintherose** [comment]
Directly

**Comment by Kiri2002** [comment]
Re-committed the pkg directly at here:
https://github.com/BioArchLinux/Packages/commit/cf9e2fa0a044aca987cebd254d6e76f39b89257d

**Pull request merged**